components/modelmeshserving/modelmeshserving.go

```diff
@@ -173,3 +182,20 @@ func (m *ModelMeshServing) ReconcileComponent(ctx context.Context,
 
 	return nil
 }
+
+func getNimManagementFlag(obj metav1.Object) string {
+	removed := string(operatorv1.Removed)
+	un, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj)
+	if err == nil {
```

Comment:
it is usually a good practice in golang to keep the happy path on the left, so

```go
if err != nil {
    return removed
}
...
```

This holds true also for the if/else below

Reply:
@lburgazzoli.... updated ✅